### ISSUE TITLE
Support for Django 3

### ISSUE THREAD

**Issue by julianklotz** [torchbox/django-libsass#35]
I’m using `django-libsass`, installed version [0.7 from PyPI](https://pypi.org/project/django-libsass/#history).

In `django_libsass.py`, there’s an [import that fails in Django 3](https://github.com/torchbox/django-libsass/blob/56d7fe539987865a042013f6746781f81882d7a3/django_libsass.py#L10):
```
from django.templatetags.static import static as django_static
```

From the [release notes of Django 3](https://docs.djangoproject.com/en/3.0/releases/3.0/): “The staticfiles and admin_static template tag libraries are removed.”
Please provide support for using django_libsass with Django 3.

**Comment by gasman** [comment]
Already fixed for the next release in d17a524fb9e17f0b2999b27c22e8753e188ec772.

**Comment by julianklotz** [comment]
Thanks for your effort!
